### ISSUE TITLE
Currently the nav title also scroll while scrolling the page

### ISSUE THREAD

**Issue by sagarchoudhary96** [hackerkid/Mind-Expanding-Books#270]
It would be good to make the navbar fixed and scroll the content as the side list navigation is all fixed so the spacing on top of it looks odd after scroll.
![Oct-17-2020 20-07-02](https://user-images.githubusercontent.com/16102594/96339787-5b7cdf00-10b4-11eb-9006-b0dbf639eafb.gif)

Should i fix it and raise the PR?

**Comment by hackerkid** [comment]
@sagarchoudhary96 I think some of the PRs that are opened already will fix that. I will let you know if it doesn't fix so that you can work on that then. For now maybe try to improve something else. Maybe create a PR to make the background color better?

**Comment by VinamraGuptaa** [comment]
Hello. Is this issue still open? Are there any other issue? I would love to contribute. Thank You

**Comment by hackerkid** [comment]
@VinamraGuptaa Yes. It's open. You can see the website at https://books.vishnuks.com/

It would be awesome if you create a PR that keeps the navbar fixed. Maybe the side menu as well? But that could be another PR.

**Comment by hackerkid** [comment]
Actually since @sagarchoudhary96  opened this issue, I would let him take over this issue if he is interested in creating a PR.

@VinamraGuptaa You can work on keeping the side menu (which shows the categories) fixed?

**Comment by sagarchoudhary96** [comment]
@VinamraGuptaa  you can take this one too ahead and fix it too along with the side menu changes.

**Comment by MohamedAli00949** [comment]
Hi, I have shown the website and noticed that 

- [ ] The header is at the end of the screen and doesn't appear on the small screens.
- [ ] The sidebar is not fixable while scrolling.

You can show my screenshots for that at the end of the comment.
So I want to fix this issue and I hope to assign it to me.

![Screenshot 2023-10-02 231824](https://github.com/hackerkid/Mind-Expanding-Books/assets/76850143/adf81613-1ae6-4bc2-85ff-ed951768a373)
![Screenshot 2023-10-02 230127](https://github.com/hackerkid/Mind-Expanding-Books/assets/76850143/acb86273-9490-4f81-8476-76aa3388dc6f)
![Screenshot 2023-10-02 230043](https://github.com/hackerkid/Mind-Expanding-Books/assets/76850143/5cfb31c0-b741-469e-86f2-afd307f4d84e)

**Comment by relaxgameing** [comment]
HI , I am new in Open Source and i think i can solve the ui issues mentioned by @MohamedAli00949 in the above conversation as it is still not been solved as i have seen the website https://mindexpandingbooks.com/
I would like to contribute to this

**Comment by anshul45-github** [comment]
Hi,
I came across this issue and would love to contribute as part of Hacktoberfest. Could you please let me know if this is still open for contributions? If so, I’d be happy to start working on it.
Looking forward to your response.
Thank you

**Comment by Yubraj977** [comment]
The issue is already fixed should't you close this issue guys
